pkg/apis/admissionregistration/v1alpha1/defaults.go

```diff
@@ -30,4 +31,8 @@ func SetDefaults_ExternalAdmissionHook(obj *admissionregistrationv1alpha1.Extern
 		policy := admissionregistrationv1alpha1.Ignore
 		obj.FailurePolicy = &policy
 	}
+	if obj.NamespaceSelector == nil {
```

Comment:
What bad thing happens if we omit this?

Reply:
`nil` selector is translated as matching nothing:
https://github.com/kubernetes/kubernetes/blob/18402f6c51e9db2379f5ed5fcb6993dab9ce2e2f/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/helpers.go#L33

 But we want the default to be matching everything.